scripts/scripts.js

```diff
@@ -156,6 +158,23 @@ export async function getLocaleBlogs() {
   return window.blogs;
 }
 
+export async function getTopicTags() {
+  if (window.blogTags) return window.blogTags;
```

Comment:
minor: since the functions is called topic tags and the you are only getting the `topic` sheet, maybe it makes sense for this one to be also `window.topicTags`, to keep it semantically correct and consistent